### PR TITLE
Add first version of the README.md

### PR DESCRIPTION
**Pull request by pachonk**
Resovles #58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
+Simply run `bin/setup` after all dependencies have been met.
 
-* How to run the test suite
+You can also run `bin/setup` to reset your instance.
 
-* Services (job queues, cache servers, search engines, etc.)
+### Running the Server
 
-* Deployment instructions
+`bundle exec rails s`
```
Comment (Exellin) [inline README.md]: Also need to run bin/webpack-dev-server in another terminal.

**Comment by Exellin** [comment]
Do we want to add the erd and meeting notes to the readme?  Or add those to the wiki and reference them in the readme?

**Comment by pachonk** [comment]
Wiki

File: README.md (hunk 1 above)
Comment (pachonk) [inline README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
-* Services (job queues, cache servers, search engines, etc.)
+### Running the Server
 
-* Deployment instructions
+`bundle exec rails s` and `bin/webkpack-dev-server` in seperate terminal panes
 
-* ...
+### Running the Tests
+
+`bin/rake` or `bundle exec rspec`
```
Comment (Exellin) [inline README.md]: might want to also mention something about rubocop here as something to make sure passes before making a pull request.

Comment (pachonk) [inline README.md]: Done

**Pull request merged**